### PR TITLE
Delta 1551 update get async status/deleteContexts endpoints documentation

### PR DESCRIPTION
**Pull request by dsavchenko-sm**
https://api-reference.smartling.com/preview/delta-1551-update-get-async-status-endpoints-documentation/#operation/getAsyncContextMatchResults

### CODE REVIEW

**Review by vbarhatov — CHANGES_REQUESTED** [review]
Discussed with Dmitry. There are some issues that have to be to fix

File: spec/openapi.yaml [hunk 1]
```diff
@@ -3123,6 +3169,83 @@ paths:
           $ref: '#/components/responses/Error429ResponseDefinition'
         500:
           $ref: '#/components/responses/Error500ResponseDefinition'
+          
+  '/context-api/v2/projects/{projectId}/processes/{processUid}':
+      get:
+        summary: Retrieve asynchronous process results
+        description: |-
+          Check the status of the async request and get the results once the request is completed
```
[the remark this answers is not in the document]
Comment (dsavchenko-sm) [reply spec/openapi.yaml]: fixed

File: spec/openapi.yaml [hunk 2]
```diff
@@ -3051,10 +3097,10 @@ paths:
       description: |-
         Check the status of the match request and get results once the match is completed.
         When the match has been completed, any newly created bindings are also returned in the response.
+      deprecated: true
```
[the remark this answers is not in the document]
Comment (dsavchenko-sm) [reply spec/openapi.yaml:3101]: added

[the remark this answers is not in the document]
Comment (vbarhatov) [reply spec/openapi.yaml:3101]: `This endpoint is deprecated and will be deleted. We recommend to use <a>Retrieve asynchronous process results</a> endpoint`

File: spec/openapi.yaml [hunk 3]
```diff
@@ -2741,6 +2741,52 @@ paths:
         500:
           $ref: '#/components/responses/Error500ResponseDefinition'
 
+  '/context-api/v2/projects/{projectId}/contexts/remove/async':
+    post:
+      summary: Delete the contexts
```
Comment (vbarhatov) [inline spec/openapi.yaml:2746]: "Delete contexts"

File: spec/openapi.yaml [hunk 4]
```diff
@@ -2741,6 +2741,52 @@ paths:
...
+      description: |-
+        Submits async operation to delete the contexts metadata and the content.
```
Comment (vbarhatov) [inline spec/openapi.yaml:2748]: "Initiates an asynchronous process to delete the contexts."

File: spec/openapi.yaml [hunk 5]
```diff
@@ -3123,6 +3170,82 @@ paths:
           $ref: '#/components/responses/Error429ResponseDefinition'
         500:
           $ref: '#/components/responses/Error500ResponseDefinition'
+          
+  '/context-api/v2/projects/{projectId}/processes/{processUid}':
+      get:
+        summary: Retrieve asynchronous process results
+        description: |-
+          Check the status of the async process and get the results once the process is completed
```
Comment (vbarhatov) [inline spec/openapi.yaml:3178]: > of the asynchronous process

File: spec/openapi.yaml [hunk 6]
```diff
@@ -3123,6 +3170,82 @@ paths:
...
+                schema:
+                  $ref: '#/components/schemas/Error404Response'
+                examples:
+                  response:
+                    value:
+                      response:
+                        code: NOT_FOUND_ERROR
+                        errors:
+                        - key: not_found
+                          message: Async process not found
```
Comment (vbarhatov) [inline spec/openapi.yaml:3244]: > Asynchronous process

File: spec/openapi.yaml [hunk 7]
```diff
@@ -3123,6 +3170,82 @@ paths:
...
+        operationId: getAsyncProcessResults
+        parameters:
+        - description: A unique project identifier. This can be found in the Smartling Dashboard under Account Settings > API.
+          in: path
+          name: projectId
+          required: true
+          schema:
+            format: uuid
+            type: string
+        - description: Identifier of the process obtained when running asynchronous request.
```
Comment (vbarhatov) [inline spec/openapi.yaml:3190]: > Identifier of the asynchronous process obtained when the process was initiated.

File: spec/openapi.yaml (hunk 4 above)
Comment (vbarhatov) [reply spec/openapi.yaml:2748]: Please update "Delete the context" endpoint and change the first sentence in the description:
> Deletes the context.

**Review by vbarhatov — CHANGES_REQUESTED** [review]
much better but a few things should be updated

**Review by MihailTymoshenko — APPROVED** [review]
👍

**Review by vbarhatov — COMMENTED** [review]
ok

**Pull request closed without merging**